python/python/tests/test_async_client.py

```diff
@@ -10254,3 +10255,155 @@ async def test_script_large_keys_and_args(self, request, cluster_mode, protocol)
             == key.encode()
         )
         await glide_client.close()
+
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_script_exists(self, glide_client: TGlideClient):
+        script1 = Script("return 'Hello'")
+        script2 = Script("return 'World'")
+
+        # Load the scripts
+        await glide_client.invoke_script(script1)
```

Comment:
Isn't it loaded when created automatically?

Reply:
Without calling `invoke_script` these 2 sha1's would return false in `script_exist` so I guess it's not being loaded by just initializing?

Reply:
Probably, I'm just asking.

Reply:
no its not we have to use invoke_script :(